samples/dockerbuilds/ARM/Dockerfile

```diff
@@ -91,6 +94,7 @@ RUN echo "SET(CURL_LIBRARIES ${TOOLCHAIN_PREFIX}/lib/libcurl.so)" >> toolchain.c
 RUN echo "SET(ENV{LDFLAGS} -L${TOOLCHAIN_PREFIX}/lib)" >> toolchain.cmake
 RUN echo "SET(set_trusted_cert_in_samples true CACHE BOOL \"Force use of TrustedCerts option\" FORCE)" >> toolchain.cmake
 RUN echo "include_directories(${TOOLCHAIN_PREFIX}/include)" >> toolchain.cmake
+RUN echo "include_directories(${TOOLCHAIN_PREFIX}/include/azureiot)" >> toolchain.cmake
```

Comment:
I don't understand: how was this working in the past? If we change the install directory, wouldn't all samples require something like `#include <azureiot/...>` before and now without azureiot?

Reply:
That's something I was not able to pinpoint.